src/mscorlib/src/System/Diagnostics/Eventing/EventPipeController.cs

```diff
@@ -94,7 +94,7 @@ internal static void Initialize()
         private EventPipeController()
         {
             // Set the config file path.
-            m_configFilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, BuildConfigFileName());
+            m_configFilePath = Path.Combine(AppContext.BaseDirectory, BuildConfigFileName());
```

Comment:
Wait, these aren't the same thing?

Reply:
Nope, they are not.  See https://github.com/dotnet/coreclr/blob/release/2.2/src/mscorlib/src/System/AppContext/AppContext.cs#L32.

```AppContext.BaseDirectory``` is implemented as:
```
return (string)AppDomain.CurrentDomain.GetData("APP_CONTEXT_BASE_DIRECTORY") ?? AppDomain.CurrentDomain.BaseDirectory;
```